### PR TITLE
they add L2 themes to coda

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

**Review by as2388 — CHANGES_REQUESTED** [review]
Why does this scheme contain yes/no/ambivalent codes? I thought we weren't using them in favour of the new style "yes, reason" codes.

**Comment by IsaackMwenda** [comment]
> Why does this scheme contain yes/no/ambivalent codes? I thought we weren't using them in favour of the new style "yes, reason" codes.

That is still the case but the question was essentially a yes/no reason so we have received many yes/no/amb responses without reasons so I suggested we add standalone yes/no/amb themes to cover those.

**Comment by as2388** [comment]
In that case can we change to something like "yes, no reason" and "no, no reason", just to be super clear that this is not a generic "yes/no" code

**Comment by IsaackMwenda** [comment]
> In that case can we change to something like "yes, no reason" and "no, no reason", just to be super clear that this is not a generic "yes/no" code

Yup, that much better. Updated thanks.

File: code_schemes/s01e01.json [hunk 1]
```diff
@@ -33,6 +33,120 @@
...
+        "VisibleInCoda": true,
+        "Shortcut": "a"
+      },
+      {
+        "DisplayText": "yes, no reason",
+        "VisibleInCoda": true,
+        "NumericValue": 1,
+        "CodeID": "code-d6a975c6",
+        "CodeType": "Normal",
+        "StringValue": "yes_no_reason"
```
Comment (as2388) [inline code_schemes/s01e01.json:52]: This might also be confusing, but can't think of anything better so ok with it RDA are happy.

**Review by as2388 — APPROVED** [review]

**Pull request merged**